package.json

```diff
@@ -7,6 +7,7 @@
     "npm": "~3.8.x"
   },
   "dependencies": {
+    "app": "^0.1.0",
```

Comment:
Yeah I think we should disable, because we haven't run into any missing library problems or anything that would be fixed by adding a dependency like this?